helpers/integration.go

```diff
@@ -476,6 +476,37 @@ func AddFinalizerToPipelineRun(adapterClient client.Client, logger IntegrationLo
 
 		logger.LogAuditEvent("Added Finalizer to the Integration PipelineRun", pipelineRun, LogActionUpdate, "finalizer", finalizer)
 	}
+	return nil
+}
+
+// AddFinalizerToComponent adds the finalizer to the component.
+// If finalizer was not added successfully, a non-nil error is returned.
+func AddFinalizerToComponent(adapterClient client.Client, logger IntegrationLogger, ctx context.Context, component *applicationapiv1alpha1.Component, finalizer string) error {
```

Comment:
We'll need [unit test coverage](https://app.codecov.io/gh/redhat-appstudio/integration-service/pull/410?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=redhat-appstudio) for these functions - should be simple enough to add them to the [existing similar ones](https://github.com/redhat-appstudio/integration-service/blob/8038e5a25bd7ce574bf1ba9ab4298ab9c801d0b2/helpers/integration_test.go#L1042-L1057).

Reply:
ack pr now includes [unit test](https://github.com/redhat-appstudio/integration-service/blob/9d75fa1ad8e2fe28fea5e9c327950e5a9f7a953b/helpers/integration_test.go#L1059), thanks for pointing me to the good reference.

Reply:
Thanks, looks good!